docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -237,7 +237,7 @@ truck,something3,buck
 |`valueColumn`|The name of the column containing the value|no|The second column|
 |`delimiter`|The delimiter in the file|no|tab (`\t`)|
 |`listDelimiter`|The list delimiter in the file|no| (`\u0001`)|
```

Comment:
Add more explanations or at least refer to some other doc page where it is explained in more details. Imagine that somebody is new to Druid and this table is the first piece of docs that he reads. I cannot even be 100% sure what does "The list delimiter in the file" mean, and "The multiValue delimiter in the file" is definitely confusing.